### PR TITLE
actions improvements

### PR DESCRIPTION
**Pull request by vdovhanych**
- **chore: add condition to run workflows only for trezor-suite repository**
- **chore: move template and params files to actions folder**
- **chore: build desktop apps on push to develop**

### CODE REVIEW

**Comment by Nodonisko** [comment]
Why is that condition necessary?

**Comment by vdovhanych** [comment]
> Why is that condition necessary?

it is not, but it is a good thing to check what repository it runs on so if somebody forks the repository, it is not trying to build and deploy stuff to our infra. It would create a lot of unnecessary errors for the forks (it already does)

**Comment by Nodonisko** [comment]
Will it affect PRs from forks too?

**Comment by vdovhanych** [comment]
> Will it affect PRs from forks too?

It shouldnt as the actions run in the repository where the pull request is created.

**Review by Nodonisko — APPROVED** [review]

**Comment by komret** [comment]
@vdovhanych please resolve the conflicts and I'll merge it.

**Comment by vdovhanych** [comment]
Should be ready. This will also mean we will have a better setup with the private release repo so we do not run the checks twice for the same stuff and will allows us to have the release actions and everything related in the public repository but reverse the condition so it never runs on the public one (even though i wont have access to secrets or runners anyway)

File: .github/workflows/crowdin_sync.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ concurrency:
 
 jobs:
   crowdin-sync:
+    if: github.repository == 'trezor/trezor-suite'
```
Comment (matejkriz) [inline .github/workflows/crowdin_sync.yml:11]: It can be useful to have crowdin-sync allowed in private repo, but it can be done manually if needed.

**Pull request merged**